### ISSUE TITLE
Project Review

### ISSUE THREAD

**Issue by alcatraz69** [tanaypratap/teamtanay.jobchallenge.dev#397]
Hey mentors,
I hope you all are doing well.
I have created my first node project. It is a Shopping list app please provide feedback.
Here's the link for the web app and git repo.

[App](https://shrouded-brook-07966.herokuapp.com/)
[Github](https://github.com/alcatraz69/List-It-Up)

Thank you.

**Comment by sandeep-devarapalli** [comment]
@alcatraz69 Nice project, are you available for hire? We need a MERN stack developer.

**Comment by alcatraz69** [comment]
> @alcatraz69 Nice project, are you available for hire? We need a MERN stack developer.

Yes, I am available for hire. I have other MERN projects to showcase!

**Comment by sandeep-devarapalli** [comment]
@alcatraz69 

> > @alcatraz69 Nice project, are you available for hire? We need a MERN stack developer.
> 
> Yes, I am available for hire. I have other MERN projects to showcase!

That's great. 
Kindly share the projects and CV at contact@hackstrap.com 
We can also discuss the commercials.
